client/src/components/users/UserCard.vue

```diff
@@ -65,6 +84,7 @@ const emits = defineEmits<{
 const props = defineProps<{
   user: UserModel;
   showCheckbox: boolean;
+  mainUser?: boolean;
```

Comment:
`isCurrentUser` instead of `mainUser`. `mainUser` implies some sort of `User` type instead of a boolean, and `user-list-display` and `user-grid-display` have a prop called `currentUser`.

Reply:
Yes, much more accurate.